pretty_midi/instrument.py

```diff
@@ -84,6 +86,16 @@ def get_piano_roll(self, fs=100, times=None):
         times : np.ndarray
             Times of the start of each column in the piano roll.
             Default ``None`` which is ``np.arange(0, get_end_time(), 1./fs)``.
+        use_pedal : Boolean
+            Control Change 64 (Sustain pedal) is reflected as elongation of
+            notes.
+            Default is False, which does nothing.  If True, then CC64 value
+            greater than pedal_threshold will be treated as pedal on and
+            the rest as pedal off.
+        pedal_threshold : Int
```

Comment:
Is there any standard for this?  Like what does it mean for a MIDI file to have a sustain pedal CC value of 1?

Reply:
Not really.  It depends more on how a synthesizer is implemented.  For example, piano-specific synthesizers interpret CC value of 1 to be a very soft pedal (like a Yamaha Disklavier), while most software synthesizers treat them as a no-pedal.  
So it's really dependent on the context of the MIDI data and the use case of this library's user.

Reply:
Ok, interesting.  Along these lines, I think the most common user preference would be for sustain pedal to be taken into consideration (corresponding to `use_pedal=True` in your code).  For that reason I'd suggest we unify `use_pedal` and `pedal_threshold` into a single argument, `pedal_threshold`, which defaults to 64.  If `pedal_threshold is None`, then we don't use the sustain pedal. What do you think?

As an aside, as a nitpick, please list types in the docstring literally, e.g. bool and int instead of Boolean and Int.